frontend/src/InteractiveImport/Series/SelectSeriesModalContent.tsx

```diff
@@ -1,30 +1,150 @@
-import React, { useCallback, useMemo, useState } from 'react';
+import { throttle } from 'lodash';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 import { useSelector } from 'react-redux';
+import { FixedSizeList as List, ListChildComponentProps } from 'react-window';
 import TextInput from 'Components/Form/TextInput';
 import Button from 'Components/Link/Button';
 import ModalBody from 'Components/Modal/ModalBody';
 import ModalContent from 'Components/Modal/ModalContent';
 import ModalFooter from 'Components/Modal/ModalFooter';
 import ModalHeader from 'Components/Modal/ModalHeader';
 import Scroller from 'Components/Scroller/Scroller';
+import Column from 'Components/Table/Column';
+import VirtualTableRowButton from 'Components/Table/VirtualTableRowButton';
 import { scrollDirections } from 'Helpers/Props';
 import Series from 'Series/Series';
 import createAllSeriesSelector from 'Store/Selectors/createAllSeriesSelector';
+import dimensions from 'Styles/Variables/dimensions';
 import translate from 'Utilities/String/translate';
+import SelectSeriesModalTableHeader from './SelectSeriesModalTableHeader';
 import SelectSeriesRow from './SelectSeriesRow';
 import styles from './SelectSeriesModalContent.css';
 
+const columns = [
+  {
+    name: 'title',
+    label: () => translate('Title'),
+    isVisible: true,
+  },
+  {
+    name: 'year',
+    label: () => translate('Year'),
+    isVisible: true,
+  },
+  {
+    name: 'imdbId',
+    label: () => translate('ImdbId'),
+    isVisible: true,
+  },
+  {
+    name: 'tvdbId',
```

Comment:
I think `tvdbId` before `imdbId` would be better since `tvdbId` will always be there